### DISCUSSION TITLE
settings are not svaing

### DISCUSSION THREAD

**Discussion by MK-HATERS** [AliDehbansiahkarbon/ChatGPTWizard#63] [Q&A]
So basically i added this plugin through delphi. download ollama, and I follow the instructions, well the settings do not save.
i fought and fought by clicking save and close then trying to chat....
now I can use codellama in cmd but idk......

**Comment by AliDehbansiahkarbon** [comment]
Hi,
Setting will be stored in the Windows registry, under this key : 
Computer\HKEY_CURRENT_USER\Software\ChatGPTWizard

Have a look, if there is not such a registry key it means your Windows user doesn't have permission to write there probably.
You or the network Administrator can create the registry key manually using the attached export to this comment too.
Of course, you just need to modify the "ChatGPTURL", and "ChatGPTModel" lines in this file(open with NotePad) before applying.

[ChatGPTWizard.zip](https://github.com/AliDehbansiahkarbon/ChatGPTWizard/files/15374111/ChatGPTWizard.zip)
